oc_mirror.sh

```diff
@@ -81,3 +78,74 @@ EOF
    popd
 
 }
+
+function set_full_imageset() {
```

Comment:
nit: maybe this name it's a little bit misleading, this function just generate a manifest file for mirroring to the local storage

Reply:
Yes agree. Changed the names of these functions.